postownik/app/client/src/App.svelte

```diff
@@ -1,9 +1,8 @@
 <script>
-  import List from './Panels/List.svelte';
-  import { authenticated } from './store.js';
-  $: console.log($authenticated);
+  import Postownik from './Panels/Postownik.svelte';
```

Comment:
Tu już ci mówiłem, żeby nazwać to `Post` zamiast Postownik jak nazwa całej apki xD